sale_product_set/demo/product_set.xml

```diff
@@ -1,11 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<openerp>
-    <data>
-        <record id="product_set_i5_computer" model="product.set">
-            <field name="name">i5 computer offer</field>
-        </record>
-        <record id="product_set_services"  model="product.set">
-            <field name="name">Services</field>
-        </record>
-    </data>
-</openerp>
+<odoo noupdate="1">
```

Comment:
Isn't a demo data convention to set noupdate="1" for demo data? preventing them to be rested into every -U